index.js

```diff
@@ -3,20 +3,26 @@
 const hof = require('hof');
 const app = require('express')();
 const churchill = require('churchill');
-
 const router = require('./lib/router');
 const serveStatic = require('./lib/serve-static');
 const sessionStore = require('./lib/sessions');
 const settings = require('./lib/settings');
 const defaults = require('./lib/defaults');
 
-const getConfig = config => Object.assign({}, defaults, config);
+const getConfig = function (options) {
+  let assigned;
+  [].slice.call(arguments).forEach((arg) => {
+    assigned = Object.assign({}, defaults, arg);
+  })
+  return assigned
+}
 
 module.exports = options => {
 
   const load = (config) => {
     config.routes.forEach((route) => {
-      app.use(router(route, config));
+      const routeConfig = Object.assign({route}, config)
+      app.use(router(routeConfig));
```

Comment:
In this case `route` is mutated in place with the values coming from `config`. Is this intentional?